### PR TITLE
Refactors workflows to hook component

### PR DESCRIPTION
**Pull request by rvsia**
**Description**

Refactors workflows component to hook component

**TODO**

- [X] component
~~test (next week)~~

@Hyperkid123

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@rvsia tests would be cool but maybe we can do it in some sort of "test sprint" next week. I guess there will be more changes to components.

**Comment by rvsia** [comment]
@Hyperkid123 Okay! :smiley:

File: src/smart-components/workflow/workflows.js [hunk 1]
```diff
@@ -24,163 +23,140 @@ const columns = [{
...
 ];
 
-class Workflows extends Component {
-    state = {
-      filteredItems: [],
-      isOpen: false,
-      filterValue: '',
-      rows: []
-    };
+const Workflows = props => {
```
Comment (Hyperkid123) [inline src/smart-components/workflow/workflows.js]: Can you add object destructing to props? Its easier to read.

File: src/smart-components/workflow/workflows.js [hunk 2]
```diff
@@ -24,163 +23,140 @@ const columns = [{
...
-                      direction="down"
-                    />
-                  </ToolbarItem>
-                </ToolbarGroup>
-              </Toolbar>
-            </LevelItem>
-          </Level>
-        </TableToolbar>
-      );
+  const onFilterChange = filterValue => setFilterValue(filterValue);
```
Comment (Hyperkid123) [inline src/smart-components/workflow/workflows.js]: I would use this directly no point of this function now. You will have to pass only the first value
```
onChange={value => setFiltervalue(value)}
```

**Comment by rvsia** [comment]
@Hyperkid123 :+1: Fixed + refactored two _if return_  (actionResolver, handleSetPage) statements to ternary operator

**Pull request merged**